Sources_v3/Controllers/ChannelController/ChannelController_Tests.swift

```diff
@@ -267,9 +267,6 @@ class ChannelController_Tests: StressTestCase {
         // Set bottom-to-top ordering
         controller.listOrdering = .bottomToTop
         
-        // Simulate `synchronize` call to apply changes
-        controller.synchronize()
-        
```

Comment:
Why this change?

Reply:
This test was written when there were no actions on `listOrdering` `didSet`.
But later we added DB observer reset on setting `listOrdering` so no need for `synchronize()` call.

Reply:
I see. Please feel free to put a comment like this to the commit body, it's hard to reason about these things without context.

Reply:
Sure, will do that next time 👌